### ISSUE TITLE
Is this reliable?

### ISSUE THREAD

**Issue by 6txri** [ibash/obsidian-new-window#2]
Very valuable thing that is missing in Obsidian. This seems like a good alternative. Is it reliable, however? Thanks a lot!

**Comment by 6txri** [comment]
I tried it and it gives me this error: 
"An error occurred while loading Obsidian. Error: @electron/remote is disabled for this WebContents. Call
require ("@electron/remote/main").enable(webContents) to enable it."

Any help would be much appreciated!

**Comment by ibash** [comment]
No, it won't really be reliable. I stopped using obsidian so I'm not going to support this, sorry!

**Comment by 6txri** [comment]
That's unfortunate. Nonetheless, thanks for the reply. 

Out of curiosity, why did you stop Obsidian and what are you using now?

**Comment by ibash** [comment]
I got frustrated with the ui bugs I was hitting. I actually ended up mostly using the obsidian vim plugin: https://github.com/epwalsh/obsidian.nvim
